anvio/kegg.py

```diff
@@ -1494,6 +1718,40 @@ def update_dict_for_genes_with_missing_annotations(self, gcids_list, hits_dict,
                                 'e_value': None,
                             }
 
+                            # add associated KEGG BRITE hierarchy information to database
+                            ortholog_brite_dict = self.kegg_modules_db.get_ortholog_brite_categorizations(knum)
```

Comment:
This is definitely my bad, because I set up the code this way - the `update_dict_for_genes_with_missing_annotations` function has a lot of the same code as `parse_kofam_hits`, and I see that it now forced you to put the same code in both places as well. Sorry :(  

I think we should make this part of the code (for adding the annotations to the dict) its own function, if possible, and simply call that function in both places. It may not be straightforward, since I'm pretty sure I already thought about doing this (and for some reason, I didn't). But I'm happy to take another look and try to fix it (unless you've already considered it and know why it is this way)

Reply:
Don't worry about it, but thanks for reminding me of this, since it was straightforward to make a new method to remove the duplication on my part.